.github/workflows/data-fetch.yml

```diff
@@ -3,16 +3,26 @@ name: Data Fetch
 on:
   schedule:
   - cron: "0 7 * * *" # Every day at midnight PDT
+  pull_request:
+    types: [labeled]
```

Comment:
Trigger on `labeled` event only to avoid unnecessary pushing a commit. Ideally this should be triggered only when working with fetching scripts.

Reply:
I’m not sure I understand the intention of this. Can you explain how you envision this working?

Reply:
I am not able to run the `fetch:team` script locally because I don't have the `ESLINT_GITHUB_TOKEN` available in the local env. So I updated this workflow to leverage it on my PR.

The workflow will be:
- Add `fetch data` label to PR.
- The workflow will be triggered and the latest data will be pushed to the PR branch. See this commit https://github.com/eslint/eslint.org/pull/285/commits/fd9b3159124a4885f54e843a86f6dfb141d9906e for example it wash pushed automatically by the workflow.

Reply:
This is useful when contributors are working on such fetch scripts and don't have access to the token locally.

Reply:
Ah, I see. I’m not sure this is necessary. I’m pretty sure all anyone needs to do is generate a token with org:read permissions. All the data is public so it should be available in the API for everyone. The only reason we use a token is because of the number of requests — I was afraid of hitting the request limit for unauthenticated API calls.

I’m also not sure that this label approach will work for external contributions that are made from forks.

Reply:
We can revert changes to this file.